configure

```diff
@@ -100,6 +101,40 @@ while [ "$TF_NEED_HDFS" == "" ]; do
   esac
 done
 
+while [ "$TF_NEED_LIBJPEG_TURBO" == "" ]; do
```

Comment:
Any additional dependencies are sources of flakiness, as a mirror ca be down, or slow to respond, etc. So it would be better to keep the dependencies minimal if we can.
In terms of the build file, @martinwicke  or @vrv can comment better on how we can make an internal build file available externally.